src/python/WMCore/REST/Main.py

```diff
@@ -217,7 +217,9 @@ def setup_server(self):
         cpconfig.update({'request.show_tracebacks': False})
         cpconfig.update({'request.methods_with_bodies': ("POST", "PUT", "DELETE")})
         _thread.stack_size(getattr(self.srvconfig, 'thread_stack_size', 128 * 1024))
-        sys.setcheckinterval(getattr(self.srvconfig, 'sys_check_interval', 10000))
+        if sys.version_info.major == 3 and sys.version_info.minor == 8:
```

Comment:
Valentin, according to the documentation:
https://docs.python.org/3.8/library/sys.html#sys.setcheckinterval

it has no effect since python 3.2.

So, I think we have two options here:
a) completely remove this check_interval hook; or
b) replace it according to what is suggested in the official documentation: https://docs.python.org/3.8/library/sys.html#sys.setswitchinterval

It's not 100% clear to me whether the unit (secs) is the same between checkinterval and switchinterval. But I'd be inclined to proceed with option b).